linera-views/src/scylla_db.rs

```diff
@@ -43,10 +43,20 @@ use thiserror::Error;
 type ScyllaDbClientPair = (Session, String);
 
 /// We limit the number of connections that can be done for tests.
-pub const TEST_SCYLLA_DB_MAX_CONCURRENT_QUERIES: usize = 1;
+#[cfg(any(test, feature = "test"))]
+const TEST_SCYLLA_DB_MAX_CONCURRENT_QUERIES: usize = 10;
```

Comment:
Thanks for the fix. Glad to see that the ScyllaDb client still works with concurrent queries

Reply:
Btw. do we still need this feature? (Do tests pass with 10000 ?)

Reply:
Yes, we still need this feature. On my Mac Laptop, it passes with 10, but not with 10000.